readthedocs/api/v2/views/model_views.py

```diff
@@ -272,6 +272,31 @@ def retrieve(self, *args, **kwargs):
                     )
         return Response(data)
 
+    @decorators.action(
+        detail=True,
+        permission_classes=[permissions.IsAdminUser],
+        methods=['post'],
+    )
+    def reset(self, request, **kwargs):
```

Comment:
I wonder if this should be a method on the model instead of the API? We can probably keep it here for now, but I could see other cases where it might be useful to have thus functionality.

Reply:
I'll move it to the model :+1: